src/main/groovy/edu/oregonstate/mist/slashcourses/db/InstructorDAO.groovy

```diff
@@ -19,6 +20,29 @@ public interface InstructorDAO extends Closeable{
               """ )
     Instructor getByInstructorID(@Bind("instructorId") Integer instructorId)
 
+    /**
+     * POST a instructor object
+     *
+     * @param lastName
+     * @param firstName
+     */
+    @SqlUpdate("""
+               INSERT INTO INSTRUCTOR (INSTRUCTOR_ID, LAST_NAME, FIRST_NAME)
+               VALUES (INSTRUCTOR_SEQ.NEXTVAL, :lastName, :firstName)
+               """)
+    void postInstructor(@Bind("lastName") String lastName,
```

Comment:
hmm, yes, I've considered this question as well. 
Because the **instructorId** and **instructor** fields are both optional in the current implementation, we can leave both of them blank, assign this course object to an existing instructor object by giving it a known instructorId, or create a new instructor object by giving new instructor details. In other words, if we want to create a new course which is taught by existing instructor, we can simply give it corresponding instructorId instead of creating a new instructor.